src/layouts/Layout.jsx

```diff
@@ -51,13 +51,22 @@ class Layout extends React.PureComponent {
                           'freeCodeCamp/assets/master/assets/' +
                           'logos/FCC-logo-white.png'
                           }
+                        style={{
+                          margin: 'auto', position: 'absolute',
+                        top: '0', left: '0', bottom: '0', right: '0'
+                      }}
                       />
                     </Link>
                   </Navbar.Brand>
                 </Navbar.Header>
               </Col>
               <Col md={ 9 } xs={ 12 }>
-                <FCCSearchBar />
+                <FCCSearchBar
+                style={{
```

Comment:
I know I started it with the `Navbar` component,  but let's do some best practice with regards to inline styles. 

Outside of this `Layout` class, declare a `style` object and move the styles from the search bar and `Navbar` into it like so
```js
const style = {
        navBar: { 
            // navBar styles here
        },
        searchBar: {
            // searchBar styles here
        }
    };
```

Then we can do
```js
<NavBar style={ style.navBar }>
```

It is more readable and also more performant with not having to create these objects on every render 👍